presentation/src/main/java/com/kdjj/presentation/view/adapter/OthersRecipeListAdapter.kt

```diff
@@ -24,24 +25,21 @@ class OthersRecipeListAdapter(
     }
 
     override fun bind(
-        holder: SingleViewTypeViewHolder<Recipe, ItemOthersRecipeBinding>,
-        item: Recipe,
+        holder: SingleViewTypeViewHolder<OthersRecipeModel, ItemOthersRecipeBinding>,
+        item: OthersRecipeModel,
     ) {
        with(holder.binding) {
-           textViewOthersItemTitle.text = item.title
-           textViewOthersItemStuffValue.text = item.stuff
-           textViewOthersItemViewCount.text = item.viewCount.toString()
-           textViewOthersItemTimeValue.text = calculateTotalTime(item)
-           textViewOthersItemUpdateTime.text = calculateUpdateTime(item)
+           recipe = item
+           executePendingBindings()
        }
     }
 }
 
-class RecipeDiffCallback : DiffUtil.ItemCallback<Recipe>() {
+class OthersRecipeModelDiffCallback : DiffUtil.ItemCallback<OthersRecipeModel>() {
 
-    override fun areItemsTheSame(oldItem: Recipe, newItem: Recipe): Boolean =
+    override fun areItemsTheSame(oldItem: OthersRecipeModel, newItem: OthersRecipeModel): Boolean =
         oldItem.recipeId == newItem.recipeId
```

Comment:
혹시 Data class 인데 recipeId로 비교하신 이유가 있으실까요?
```suggestion
    override fun areItemsTheSame(oldItem: OthersRecipeModel, newItem: OthersRecipeModel): Boolean =
        oldItem == newItem
```

Reply:
recipeId 가 key 니까 저것만 비교하면 된다고 생각했습니다 !